### DISCUSSION TITLE
Pass all querystring params to a redirect

### DISCUSSION THREAD

**Discussion by letto4135** [rwf2/Rocket#2251] [Questions] [answered]
Is there a way to easily pass all querystring params to a redirect? I don't care what they are, the underlying service will take care of that I just need to pass them along but can't find a simple way to do it.

**Comment by the10thWiz** [comment, marked as answer]
From a security perspective, this is likely not a good idea, which is why Rocket makes it more difficult.

The guide has a section on collecting the remaining query parameters: https://rocket.rs/v0.5-rc/guide/requests/#trailing-parameter. For your case, you likely want to just collect them into a HashMap, and write them to a url. However, if you would like to do any kind of validation against these before passing them on, I recommend creating a custom struct that implements `FromForm` with all of the fields to handle validation.
